typings/karma-fixtures.d.ts

```diff
@@ -98,133 +98,132 @@ interface RuleDescription {
 
 interface RuleTemplates {
   AlgebraicExpressionInput: {
-    MatchesExactlyWith: IRuleDescription;
-    IsEquivalentTo: IRuleDescription;
+    MatchesExactlyWith: RuleDescription;
```

Comment:
Is this change expected? \cc @nishantwrp to check as well.

Reply:
Yup it is, see https://github.com/oppia/oppia/pull/10189#discussion_r466926818

But @nishantwrp should follow up on that thread, too (though the action item is on his end).

Reply:
Ah ok, Thanks!